### PR TITLE
[lint fixes] Removed redundant parentheses and fix struct initialization without field names

### PR DESCRIPTION
**Pull request by SamuelMarks**
Fixing some lint issues

### CODE REVIEW

File: eth/handler.go [hunk 1]
```diff
@@ -392,7 +392,7 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 					unknown = true
 				} else {
 					query.Origin.Hash, query.Origin.Number = pm.blockchain.GetAncestor(query.Origin.Hash, query.Origin.Number, ancestor, &maxNonCanonical)
-					unknown = (query.Origin.Hash == common.Hash{})
+					unknown = query.Origin.Hash == common.Hash{}
```
Comment (karalabe) [inline eth/handler.go]: Whilst the changes are generally ok, could be revert these specific type of changes where `var1 = var2 == var3`. Removing the parentheses only makes the code harder to read.

File: les/benchmark.go [hunk 2]
```diff
@@ -135,7 +135,7 @@ func (b *benchmarkHelperTrie) init(pm *ProtocolManager, count int) error {
 		b.sectionCount, b.headNum, _ = pm.server.bloomTrieIndexer.Sections()
 	} else {
 		b.sectionCount, _, _ = pm.server.chtIndexer.Sections()
-		b.sectionCount /= (params.CHTFrequencyClient / params.CHTFrequencyServer)
+		b.sectionCount /= params.CHTFrequencyClient / params.CHTFrequencyServer
```
Comment (karalabe) [inline les/benchmark.go]: Whilst the changes are generally ok, could be revert these specific type of changes where `var1 op= var2 op var3`. Removing the parentheses only makes the code harder to read.

File: cmd/clef/main.go [hunk 3]
```diff
@@ -679,21 +679,21 @@ func testExternalUI(api *core.SignerAPI) {
...
-			types.Bloom{},
-			big.NewInt(1337),
-			big.NewInt(1337),
-			1338,
-			1338,
-			big.NewInt(1338),
-			[]byte("Extra data Extra data Extra data  Extra data  Extra data  Extra data  Extra data Extra data"),
-			common.HexToHash("0x0000H45H"),
-			types.BlockNonce{},
+			ParentHash: common.HexToHash("0000H45H"),
```
Comment (gballet) [inline cmd/clef/main.go]: could you please run `gofmt` on this one?

**Comment by fjl** [comment]
This PR is now out of date. Please resubmit if you still want this change.

**Pull request closed without merging**